### ISSUE TITLE
Support pacman based distros such as Arch/Manjaro in the install script

### ISSUE THREAD

**Issue by jjnp** [openfaas/faasd#173]
## Expected Behaviour

When installing faasd using the install srcript at `hack/install.sh` on a Linux distribution that uses `pacman` as the default package manager the installation should succeed.
## Current Behaviour

Currently the script terminated with an error message saying that it cannot install on this OS, because neither `apt-get` or `yum` could be found.
## List all Possible Solutions

1. Add a check to the script if `pacman` is present, and if yes simply use that instead of `yum` or `apt-get`.
2. Instead of installing the dependencies via a package manager, install manually (not recommended).

## List the one solution that you would recommend

1. Add a check to the script if `pacman` is present, and if yes simply use that instead of `yum` or `apt-get`.
## Steps to Reproduce (for bugs)

1. Try to run `./hack/install.sh` on a distro using pacman as package manager (e.g. Arch/Manjaro) i.e. a distro that has neither `apt-get` nor `yum` installed

## Context

I was trying to set up OpenFaaS with `faasd` as a backend using the `hack/install.sh` script in order to learn about OpenFaaS.
## Your Environment

* OS and architecture: Manjaro Linux, x86_64

* Versions: Manjaro Linux 21.0 (rolling release), Kernel 5.11.6-1-MANJARO

```sh
go version
go version go1.16.2 linux/amd64

containerd -version
containerd github.com/containerd/containerd v1.3.7 8fba4e9a7d01810a393d5d25a3621dc101981175
uname -a
Linux jp-manjaro 5.11.6-1-MANJARO #1 SMP PREEMPT Thu Mar 11 19:05:51 UTC 2021 x86_64 GNU/Linux

cat /etc/os-release
NAME="Manjaro Linux"
ID=manjaro
ID_LIKE=arch
BUILD_ID=rolling
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="32;1;24;144;200"
HOME_URL="https://manjaro.org/"
DOCUMENTATION_URL="https://wiki.manjaro.org/"
SUPPORT_URL="https://manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
LOGO=manjarolinux

faasd version
  __                     _ 
 / _| __ _  __ _ ___  __| |
| |_ / _` |/ _` / __|/ _` |
|  _| (_| | (_| \__ \ (_| |
|_|  \__,_|\__,_|___/\__,_|

faasd version: 0.11.4	commit: dca036ee51a7275389bf45c7839d22d437663a8e
```

**Comment by jjnp** [comment]
Full disclosure: I already applied the recommended changes. A PR is coming as soon as I get around to formatting it according to the guidelines.

**Comment by ksclarke** [comment]
Looks like this was merged, should this issue be closed?

**Comment by rgee0** [comment]
Derek close: addressed in #173
